### DISCUSSION TITLE
日志文件默认存放在哪?

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [AlistGo/alist#5759] [Q&A]
通过`./alist start`运行的时候, 是否会产生额外的日志文件或者别的什么文件?

**Comment by ririye** [comment]
在alist.exe所在目录下的data\log里面

**Reply by ririye** [reply]
data文件夹下的config.json里写的啊

**Comment by xuchaoxin1375** [comment]
https://alist.nn.ci/zh/config/configuration.html#log
